index.html

```diff
@@ -300,24 +302,41 @@ <h2>Our objective is to build quality software that performs well, improves user
   <footer>
     <ol class='links'>
       <li>
-        <a href='https://github.com/betterthingsdigital'>
+        <a href='https://github.com/betterthingsdigital' target='_blank'>
```

Comment:
most people (including me) find this behavior annoying 

// https://css-tricks.com/use-target_blank/